Source/objdat.cpp

```diff
@@ -275,7 +275,7 @@ const ObjectData AllObjects[109] = {
 /*OBJ_URN*/          { OFILE_URN,           21,      24, DTYPE_NONE,      THEME_NONE,              Q_INVALID,  Solid | MissilesPassThrough | Light | Breakable,          1,        9,         96,        3 },
 /*OBJ_URNEX*/        { OFILE_URNEX,         21,      24, DTYPE_NONE,      THEME_NONE,              Q_INVALID,  Solid | MissilesPassThrough | Light | Breakable,          1,       10,         96,        3 },
 /*OBJ_L5BOOKS*/      { OFILE_L5BOOKS,       21,      24, DTYPE_NONE,      THEME_NONE,              Q_INVALID,  Solid | MissilesPassThrough | Light,                      1,        0,         96,        3 },
-/*OBJ_L5CANDLE*/     { OFILE_L5CANDLE,      21,      23, DTYPE_NONE,      THEME_NONE,              Q_INVALID,  Animated | Solid | MissilesPassThrough | Light,           2,        4,         96,        0 },
+/*OBJ_L5CANDLE*/     { OFILE_L5CANDLE,      21,      23, DTYPE_NONE,      THEME_NONE,              Q_INVALID,  Animated | Solid | MissilesPassThrough,                   2,        4,         96,        0 },

```

Comment:
These emit a very faint light and should not be fully lit.
```suggestion
/*OBJ_L5CANDLE*/     { OFILE_L5CANDLE,      21,      23, DTYPE_NONE,      THEME_NONE,              Q_INVALID,  Animated | Solid | MissilesPassThrough | Light,                   2,        4,         96,        0 },

```